horace_core/sqw/file_io/@sqw_binfile_common/get_sqw.m

```diff
@@ -66,7 +66,29 @@
 % Get detector parameters
 % -----------------------
 if ~(opts.head||opts.his)
-    sqw_struc.detpar = obj.get_detpar();
+    detpar = obj.get_detpar();
+    if (~isempty(detpar)                             && ...
+        IX_detector_array.check_detpar_parms(detpar)    ...
```

Comment:
Do we want to assign this `detpar` if it fails `check_detpar_parms` or should that error?

Reply:
I appear to have done this